.gitignore

```diff
@@ -0,0 +1,12 @@
+# Logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
```

Comment:
What exactly is this .gitignore file for? I have one in my PR, it got created when I deployed the app.

Reply:
The server uses yarn to install modules. These means we need to ignore things like node_modules, etc.

Reply:
oh cool